### PR TITLE
Add tests and CI

### PR DESCRIPTION
**Pull request by nexdrew**
Granted the tests aren't _super_ useful since they really just test `handlebars` expansion, but they at least give some sense of assurance that the templates work.

Anyway, it was fun to write the tests in pseudo-ES7 using `ava`.

Also using `standard` style for test code.

### CODE REVIEW

File: test/charge-failed.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+test('expansion of meta', async t => {
+  const template = await loadTemplate('charge-failed.meta.hbs')
+  const metaString = template({
+    name: 'Exquisite Disaster',
+    email: 'nunya@biznazz.yo',
+    from: 'website@npmjs.com'
+  })
+  const meta = JSON.parse(metaString)
+  t.is(meta.subject, 'npm, Inc.: charge failed')
+  t.is(meta.to, '"Exquisite Disaster" <nunya@biznazz.yo>')
```
Comment (ashleygwilliams) [inline test/charge-failed.js]: :laughing:

**Comment by ashleygwilliams** [comment]
:clap: :clap: :clap: :clap: :clap: :clap: :clap: :clap:

**Comment by bcoe** [comment]
@nexdrew any reason not to get code coverage in the mix with `nyc` :smile: -- curious if that's working well with AVA.

**Comment by nexdrew** [comment]
@bcoe Well in this repo there is no code to cover :smile: Besides tests, it's only templates.

But I do know that Sindre is using `nyc` with AVA, e.g. in https://github.com/sindresorhus/got, so I think you're good there. :+1:

**Comment by bcoe** [comment]
@nexdrew always so logical! :stuck_out_tongue:

**Pull request merged**